### PR TITLE
testcase/ta_tc/audio/utc : Modify Test Log Format

### PR DESCRIPTION
**Pull request by YiHyunJin**
Modify Test Log Format

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c:812: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)
apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c:812: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)

**Comment by seinfra** [comment]
Code Rule Check Result:
apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c:812: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)
apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c:812: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)

File: apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c [hunk 1]
```diff
@@ -807,10 +809,7 @@ static int audio_tc_launcher(int argc, char **args)
 	/* after test, unlink the file */
 	unlink(AUDIO_TEST_FILE);
 
-	printf("#########################################\n");
-	printf("           Audio TC Result               \n");
-	printf("           PASS : %d FAIL : %d        \n", total_pass, total_fail);
-	printf("#########################################\n");
+	printf("########## Audio UTC End [PASS : %d, FAIL : %d] ##########\n",total_pass,total_fail);
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c]: add space after ','

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**